packages/fast-usdc/src/exos/status-manager.js

```diff
@@ -236,7 +245,13 @@ export const prepareStatusManager = zone => {
        */
       forwarded(txHash, address, amount) {
         // TODO: store txHash -> evidence for txs pending settlement?
-        console.log('TODO: vstorage update', { txHash, address, amount });
+        if (txHash) {
+          recordStatus(txHash, 'FORWARDED');
```

Comment:
```suggestion
          recordStatus(txHash, TxStatus.Forwarded);
```

Reply:
Please also remove L247: `// TODO: store txHash -> evidence for txs pending settlement?`

Reply:
Done